ci/doc.sh

```diff
@@ -13,6 +13,13 @@ set -euo pipefail
     ../doc/argparse2rst.py ./*.py > ../doc/_gen/utils_py.rst
 )
 
+(
+    cd ./doc
+    if [ ! -e notebook ]; then
+        git clone https://github.com/espnet/notebook
```

Comment:
To support off-line doc build in the future, we may consider adding this dependency as a git submodule, which is more explicit and sounds reasonable approach to me.

Reply:
Of course. I would like to use `git checkout xxxx` instead of git submodule (as well as `tools/Makefile`) because it is too difficult for us... (sorry)

Reply:
FYI I used to use git submodule to include notebooks to sphinx doc (see https://github.com/r9y9/nnmnkwii/tree/master/docs) and I get no problem so far. Off course I'm okay with git checkout though :)

Reply:
Thanks. We decided to use git submodule!